sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/src/autorest.md

```diff
@@ -0,0 +1,24 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+```yaml
+directive:
+- from: swagger-document
+  where: $.definitions.*
+  transform: >
+    $["x-csharp-usage"] = "model,input,output";
```

Comment:
This shouldn't be necessary. Our generate will infer this in almost every case. We don't use it for any of our other SDKs. Nor is the line below.